.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Emscripten/Pyodide build
+
+on:
+  push:
+    branches:
+      - emscripten-ci
```

Comment:
I guess we can leave this in as a convenient way to trigger a build on a fork perhaps, but not sure that that is intended?

Reply:
This was intended as a temporary thing, will revert before merging.